azure-pipelines.yml

```diff
@@ -0,0 +1,77 @@
+name: TL-Marketing-and-Communications
```

Comment:
```suggestion
name: tl-marketing-and-communications
```

Reply:
Convention is normally lowercase

Reply:
That name field is build number format, so this is incorrect either way, it'll be set by Gitversion